core/src/main/java/com/netease/arctic/utils/SchemaUtil.java

```diff
@@ -67,4 +69,21 @@ public static Schema fillUpIdentifierFields(Schema baseSchema, Schema fromSchema
 
     return new Schema(schemaId, fields, identifierFieldIds);
   }
+
+  /**
+   * Convert an Iceberg Schema {@link Schema} to a {@link Schema} based on the given fieldNames.
+   *
+   * @param baseSchema a Schema on which loading is based
+   * @param fieldNames a list of field names
+   * @return a new Schema on which contain the fieldNames of the base schema.
+   */
+  public static Schema convertFieldsToSchema(Schema baseSchema, List<String> fieldNames) {
```

Comment:
You can use org.apache.iceberg.Schema#select(java.lang.String...)

Reply:
Good point.